src/commcare_cloud/commands/terraform/aws.py

```diff
@@ -343,6 +347,18 @@ def aws_sign_in(environment, duration_minutes=DEFAULT_SIGN_IN_DURATION_MINUTES,
                                      force_new=force_new)
 
 
+def is_ec2_instance_in_account(account_id):
+    try:
+        aws_instance_identity_doc = requests.get(
+            'http://169.254.169.254/latest/dynamic/instance-identity/document',
```

Comment:
how come this IP is hard coded? Is it a global AWS endpoint?

Reply:
It's an AWS EC2 instance thing described here https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-identity-documents.html. But basically that IP address is available from EC2 instances but returns information specific to the EC2 instance you are requesting it from. To quote that link
> The IP address 169.254.169.254 is a link-local address and is valid only from the instance.

Sorry for the very late reply!